deploy-manage/security/set-up-basic-security-plus-https.md

```diff
@@ -1,48 +1,65 @@
 ---
-navigation_title: "Set up basic security plus HTTPS"
+navigation_title: "Set up HTTPS"
 applies_to:
   deployment:
     self: ga
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/security-basic-setup-https.html
 ---
 
+% Scope: HTTP certificates setup / manual configuration / multi or single node cluster
+% original title: Set up basic security for the Elastic Stack plus secured HTTPS traffic
+# Set up HTTPS [security-basic-setup-https]
 
+Enabling TLS on the HTTP layer, widely known as HTTPS, ensures that all client communications with your cluster are encrypted, adding a critical layer of security.
 
-# Set up basic security plus HTTPS [security-basic-setup-https]
+This document focuses on the **manual configuration** of HTTPS for {{es}} and {{kib}}. Use this approach if you want to provide your own TLS certificates, generate them with Elastic’s tools, or have full control over the configuration. Alternatively, {{es}} can [automatically generate and configure HTTPS certificates](./self-auto-setup.md).
```

Comment:
can we specify which of the steps in this guide are not automated (the kibana ones)?

Reply:
Updated the sentence to:

Note that HTTPS configuration for {{kib}} is always manual. Alternatively, {{es}} supports [automatic HTTPS setup](./self-auto-setup.md), which can simplify the process if full customization isn't required.

Hoping that clarifies all.